manim/mobject/three_dimensions.py

```diff
@@ -134,6 +134,28 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
 
 
 class Sphere(ParametricSurface):
+    """
```

Comment:
Just as mentioned in the other comment: every docstring **needs** to start with a one-line summary. Please add back something along the lines of

```suggestion
    """A mobject representing a three-dimensional sphere.
```

(feel free to commit your own suggestion instead.)